FSharpLu.Json/Scripts/load-project-debug.fsx

```diff
@@ -0,0 +1,9 @@
+// Warning: generated file; your changes could be lost when a new file is generated.
```

Comment:
We don't check-in those files since they can be auto-generated. You can add them to .gitignore.